src/backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+#base image
+FROM python:3.11
+# set work directory
+WORKDIR /app
+
+# installation of dependencies
+COPY requirements.txt /app/requirements.txt
+RUN pip install -r requirements.txt
+
+# copy project
```

Comment:
Comments in Line 10 and Line 12 are not necessary here. The code is self-explained

Reply:
how about keep them for now, and we could delete all of dead code and unnecessary comments when the project is almost down?